src/status_im2/subs/communities_test.cljs

```diff
@@ -109,3 +109,24 @@
             "0x3" {:id "0x3" :name "Civilized dolphins"}})
     (is (= {:joined ["0x1"], :pending ["0x2"], :opened ["0x3"]}
            (rf/sub [sub-name])))))
+
+(h/deftest-sub :communities/sorted-communities
```

Comment:
@ilmotta added the sorted communities subscription and this test. It's alright? :)

Reply:
Looks like a useful test @J-Son89!

Reply:
thanks for checking. and for helping set it up. Had not looked at these tests before and it was easy to figure out how to use it from the other tests 👍